scripts/vertagen/README.md

```diff
@@ -0,0 +1,44 @@
+# Vertagen 
```

Comment:
Since this is used across multiple CaraML projects, should we add vertagen as a separate Gojek repo and link to the others using a submodule?

Reply:
I'd rather avoid using git submodule for this scenario. I was thinking about packaging `vertagen` as a Github action to share between the workflows, but then it will also require some other approach for the local dev environment.